metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfigDescriptor.java

```diff
@@ -82,6 +82,11 @@ private void loadProps() {
       logger.warn("Fail to find config file, use default");
     }
     metricConfig = new MetricConfig();
+    if (metricConfig.getEnableMetric()) {
+      logger.warn("metricConfig enable");
+    } else {
+      logger.warn("metricConfig not enable");
+    }
```

Comment:
I think this log level shouldn't be warn? maybe debug or info.

Reply:
OK, these logs don't seem important. I've removed them.